CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
```

Comment:
Can we add an "Unreleased" section for future features?
```suggestion

## Unreleased

```

If you want to keep the CHANGELOG clean for the release I'm okay with that too.

Reply:
Yeah. I would prefer keeping the changelog clean for this one. Can think about having the "unreleased" section in the future.